keras_cv/metrics/coco/recall.py

```diff
@@ -152,9 +152,9 @@ def update_state(self, y_true, y_pred, sample_weight=None):
                 y_true_for_image = utils.filter_boxes_by_area_range(
                     y_true_for_image, self.area_range[0], self.area_range[1]
                 )
-                y_pred_for_image = utils.filter_boxes_by_area_range(
-                    y_pred_for_image, self.area_range[0], self.area_range[1]
-                )
+                # y_pred_for_image = utils.filter_boxes_by_area_range(
```

Comment:
Any reason this is commented out?

Reply:
Ah that was a test - good catch.  Shouldn't have been there.